manual/src/cmds/intf-c.etex

```diff
@@ -2434,6 +2434,14 @@ is held, therefore the C code has full access to the facilities of the
 run-time system.  However, no other thread in the same domain can execute
 OCaml code concurrently with the C code of the primitive.
 
+This implies that C code running in other domains *can* run in parallel
```

Comment:
How about 

> During parallel execution with multiple domains, C code running on a domain may run in parallel with any C code running in other domains even if neither of them has released the ``domain lock''. Prior to OCaml 5.0, C bindings may have assumed that if the OCaml runtime lock is not released, then it would be safe to manipulate global C state (e.g. initialise a function-local `static` value). This is no longer true in the presence of parallel execution with multiple domains.